profile/README.md

```diff
@@ -1 +1,8 @@
-Welcome!
+## Welcome to the OpenVINO™ GitHub organization!
+[OpenVINO™](https://en.wikipedia.org/wiki/OpenVINO) is an open-source toolkit for optimizing and deploying AI inference.
+* [openvino](https://github.com/openvinotoolkit/openvino) is the main repository with the sourse code for runtime and some tools.
+* [nncf](https://github.com/openvinotoolkit/nncf) Neural Network Compression Framework is used for enhanced OpenVINO™ inference to get performance boost with minimal accuracy drop.
+* [openvino_notebooks](https://github.com/openvinotoolkit/openvino_notebooks) repository contains Jupyter notebook tutorials to demonstrate key features of the toolkit.
+
+See [CONTRIBUTING](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md) for contribution details. <br>
+Get support using: The [`openvino`](https://stackoverflow.com/questions/tagged/openvino) tag on StackOverflow\* or [Forum](https://software.intel.com/en-us/forums/computer-vision)
```

Comment:
something is wrong with the forum link access (at least for me)
this is the one used in the docs https://community.intel.com/t5/Intel-Distribution-of-OpenVINO/bd-p/distribution-openvino-toolkit